lib/govuk_index/publishing_event_worker.rb

```diff
@@ -47,7 +48,11 @@ def process_action(actions, routing_key, payload)
       Services.statsd_client.increment('govuk_index.sidekiq-consumed')
 
       document_type_inferer = DocumentTypeInferer.new(payload)
-      presenter = ElasticsearchPresenter.new(payload, document_type_inferer.type)
+      presenter = ElasticsearchPresenter.new(
+        payload: payload,
+        type: document_type_inferer.type,
+        sanitiser: IndexableContentSanitiser.new(payload)
```

Comment:
as discussed can we pass the payload into the clean method instead of the initializer